osu.Game/Screens/Select/Carousel/SetPanelBackground.cs

```diff
@@ -17,13 +17,12 @@ namespace osu.Game.Screens.Select.Carousel
     public partial class SetPanelBackground : BufferedContainer
     {
         public SetPanelBackground(IWorkingBeatmap working)
-            : base(cachedFrameBuffer: true)
```

Comment:
I had to disable this to make the effect work. I'm not 100% sure what the purpose of it was (my guess is performance), so I'm not sure if this would have a negative effect.
If it was for performance it might be a good idea to use a regular container instead here since it now just adds an extra render to a framebuffer per frame per item.

Reply:
Yes, it's for performance (dealing with huge textures, downsampling, multiple overlay layers).